spec/controllers/verify/sessions_controller_spec.rb

```diff
@@ -191,6 +192,20 @@
           expect(response).to redirect_to verify_fail_url
         end
       end
+
+      context 'attempt window has expired, previous attempts == max-1' do
+        before do
+          user.idv_attempts = max_attempts - 1
+          user.idv_attempted_at = Time.zone.now - 2.days
+        end
+
+        it 'allows and resets attempt counter' do
+          post :create, profile: user_attrs
+
+          expect(response).to redirect_to verify_finance_path
+          expect(user.idv_attempts).to eq 1
+        end
+      end
```

Comment:
Can we also add this same test in the other IdV controllers? I believe that if we added this test to the other controllers, they would fail if this PR was left as is.